getcord/spr
# When updating a PR and no --message is given, prompt user

Test Plan:
* submit new PR - message for the commit should be "Initial version"
* update PR using "--message" option
* update PR without "--message" option
